test/runtests.jl

```diff
@@ -133,17 +155,91 @@ end
 
 const testdir = dirname(@__FILE__)
 @testset for test in tests
+    yamlString = open(joinpath(testdir, string(test, ".data"))) do f
+        read(f, String)
+    end
+
+    expected = evalfile(joinpath(testdir, string(test, ".expected")))
+
+    # Test Loading File with Constructor Object
     data = YAML.load_file(
         joinpath(testdir, string(test, ".data")),
-        more_constructors
+        TestConstructor()
     )
-    expected = evalfile(joinpath(testdir, string(test, ".expected")))
     @test equivalent(data, expected)
+    dictData = YAML.load_file(
+        joinpath(testdir, string(test, ".data")),
+        more_constructors, multi_constructors
+    )
+    @test equivalent(dictData, expected)
+
+    stringData = YAML.load(yamlString, TestConstructor())
+    @test equivalent(stringData, expected)
+    dictStringData = YAML.load(yamlString, more_constructors, multi_constructors)
+    @test equivalent(dictStringData, expected)
+
+
+    @test equivalent(first(YAML.load_all_file(joinpath(testdir, string(test, ".data")), TestConstructor())), expected)
+    @test equivalent(first(YAML.load_all_file(joinpath(testdir, string(test, ".data")), more_constructors, multi_constructors)), expected)
+
+    @test equivalent(first(YAML.load_all(yamlString, TestConstructor())), expected)
```

Comment:
Failing on Windows

Reply:
Code works when manual running code on Windows 10, build 2004 and Julia 1.4.2